Formula/t/tailwindcss.rb

```diff
@@ -17,7 +22,19 @@ class Tailwindcss < Formula
 
   depends_on "node"
 
+  resource "tailwind-cli" do
+    url "https://registry.npmjs.org/@tailwindcss/cli/-/cli-4.0.0.tgz"
+    sha256 "a6a772944d048966e9db2bdc7521053ea3d8bd06cfdff7931fdc4bb2313e6369"
+  end
```

Comment:
```suggestion
  resource "tailwind-cli" do
    url "https://registry.npmjs.org/@tailwindcss/cli/-/cli-4.0.0.tgz"
    sha256 "a6a772944d048966e9db2bdc7521053ea3d8bd06cfdff7931fdc4bb2313e6369"
    
    livecheck do
      formula :parent
    end
  end
```
If this resource's version should match the formula version (as it appears to be), we should add this `livecheck` block so it's automatically updated by `bump-formula-pr` (see Homebrew/brew#19130).

Reply:
Hm, then the related test in the `install` block should be removed if we're not sure, i.e., this line:
```
odie "tailwind-cli resource needs to be updated" if version != resource("tailwind-cli").version
```